tests/subsys/tracing/tracing_api/src/main.c

```diff
@@ -179,57 +153,25 @@ void test_tracing_sys_api(void)
 	sys_trace_k_timer_init(&timer, NULL, NULL);
 
 	/* wait for some actions finished */
-	k_sleep(K_MSEC(100));
-	zassert_true(tracing_api_found, "Failded to check output from backend");
-	zassert_true(tracing_api_not_found == false, "Failded to check output from backend");
-}
-#else
-/**
- * @brief Test tracing APIS
- *
- * @details For synchronize mode, self-designed tracing uart backend
- * firstly, called API tracing_format_string to put a string, meanwhile,
- * check the output for the backend.
- *
- * @ingroup tracing_api_tests
- */
-void test_tracing_sys_api(void)
-{
-	tracing_buffer_init();
-	tracing_format_string("tracing_format_string_testing");
-	k_sleep(K_MSEC(100));
+	k_sleep(K_MSEC(500));
```

Comment:
Seems like the delay time increasing here. Is there any reason for that?

Reply:
Hi @enjiamai,  the reason why I added the delay here is that I need to make sure the data has been finished passing by tracing before the test case verifies the flags.